.github/scripts/common-test.mk

```diff
@@ -0,0 +1,143 @@
+CPUNAME ?= cpu1
+INSTALL_DIR ?= $(O)/exe
+
+ALL_OSAL_FUNC_TESTS :=          	\
+    bin-sem-flush-test		\
+    bin-sem-test			\
+    bin-sem-timeout-test	\
+    count-sem-test			\
```

Comment:
This is missing count-sem-timeout-test, and shell-test.

Is there a way to not need this to be hardcoded. Similar to how the CMake file for the tests works. 

https://github.com/nasa/osal/blob/f11d04961d0410af1adb6a2202549b195be3fa5a/src/tests/CMakeLists.txt#L11-L16